best-practices/massive-regions-best-practices.md

```diff
@@ -134,7 +134,11 @@ config set merge-schedule-limit 8
 
 ### 方法六：调整 Region 大小
 
-Region 默认的大小约为 96 MiB，将其调大也可以减少 Region 个数，具体介绍可参考[使用更大的 Region](/tune-region-performance.md)。
+Region 默认的大小约为 256 MiB，将其调大也可以减少 Region 个数，具体介绍可参考[使用更大的 Region](/tune-region-performance.md)。
+
+> **注意：**
+>
+> 从 v8.4.0 开始，Region 的默认值从 96 MiB 调整为 256 MiB。
```

Comment:
For deployments upgraded from a prior version, is there anything to be aware of?